ci/test-install/CMakeLists.txt

```diff
@@ -30,5 +30,14 @@ find_package(Threads REQUIRED)
 find_package(bigtable_client 0.3.0 REQUIRED)
 
 # Defining new targets should be easy.
-add_executable(install_integration_test install_integration_test.cc)
-target_link_libraries(install_integration_test bigtable::client)
+add_executable(bigtable_install_test bigtable_install_test.cc)
+target_link_libraries(bigtable_install_test bigtable::client)
+
+find_package(CURL REQUIRED)
+if (NOT TARGET CURL::CURL)
```

Comment:
Does this empty if block do anything useful? Or was it just left in by accident?

Reply:
Accident, I started and then realized there was a better fix. Thanks for catching this.